elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -629,12 +629,22 @@ def _cc_pipeline(self,
                         output_node_parameter_key = property_value['option'].replace("elyra_output_", "")
                         operation.component_params[component_property.ref] = \
                             target_ops[output_node_id].outputs[output_node_parameter_key]
-                    elif component_property.data_type == 'dictionary':
-                        processed_value = self._process_dictionary_value(property_value)
-                        operation.component_params[component_property.ref] = processed_value
-                    elif component_property.data_type == 'list':
-                        processed_value = self._process_list_value(property_value)
-                        operation.component_params[component_property.ref] = processed_value
+                    elif component_property.data_type == "inputvalue":
+                        active_property = property_value['activeControl']
+                        active_property_value = property_value[active_property]
+                        if isinstance(property_value, dict) and set(property_value.keys()) == {'value', 'option'}:
+                            output_node_id = property_value['value']
+                            output_node_parameter_key = property_value['option'].replace("elyra_output_", "")
+                            operation.component_params[component_property.ref] = \
+                                target_ops[output_node_id].outputs[output_node_parameter_key]
+                        elif component_property.default_data_type == 'dictionary':
+                            processed_value = self._process_dictionary_value(active_property_value)
+                            operation.component_params[component_property.ref] = processed_value
+                        elif component_property.default_data_type == 'list':
+                            processed_value = self._process_list_value(active_property_value)
+                            operation.component_params[component_property.ref] = processed_value
+                        else:
+                            operation.component_params[component_property.ref] = active_property_value
```

Comment:
Are each of these statements over-indented or does only the last `else` statement need to be duplicated for the outer `if`-block?  If neither, then perhaps the `elif` on L632 should be an `else`.

Reply:
I should add an outer if block `else` to clarify that it would catch the last case which would be `outputPath` parameters. We don't do anything with these parameters right now. 
The statements should all be indented properly in that they should only apply to any field designated an `inputvalue`